functions.go

```diff
@@ -200,7 +200,7 @@ func defaultPrintCommandHelp(a *App, cmd *Command, shell bool) {
 func headlinePrinter(a *App) func(v ...interface{}) (int, error) {
 	if a.config.NoColor || a.config.HelpHeadlineColor == nil {
 		return func(v ...interface{}) (int, error) {
```

Comment:
Instead of wrapping a.Println in a func(v ...interface{}) (int, error) here, you can directly return a.Println!

Reply:
Yup, that was a stupid one.  Fixing it now.  :)